### DISCUSSION TITLE
【2023.6.8】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#430] [meeting]
**Conference link：**
> **Note**: Every two weeks at 9:00 p.m
https://meeting.tencent.com/dm/y6D9zgQbcXmF 📱#Tencent Conference：333-9699-4866

**Outline of the meeting:**
- [user case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md)
- [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- External developers attract this, and maintainer management planning needs to let external developers know
- Iterations in functionality build barriers that require more features to attract developers.
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**If you have a topic to discuss, please comment below.**

**Comment by skiffer-git** [comment]
errcode branch: Take a certain interface as an example to explain the whole process of api->rpc->db, which is convenient for developers to familiarize themselves with the architecture, secondary development, and contribute code

**Comment by cubxxw** [comment]
**gitlint:**
+ https://www.gitlink.org.cn/glcc/2023/subjects/detail/539
+ https://www.gitlink.org.cn/glcc/2023/subjects/detail/540

**Comment by IRONICBo** [comment]
About gitlink project: https://github.com/OpenIMSDK/Open-IM-Server/issues/433

**Comment by cubxxw** [comment]
[Feature:Deploy Server-side Through a Visual Web Interface.#420](https://github.com/OpenIMSDK/Open-IM-Server/issues/420)

**Comment by cubxxw** [comment]
[Feature: jssdk server optimize#422](https://github.com/OpenIMSDK/Open-IM-Server/issues/422)
